src/Symfony/Component/Form/Resources/translations/validators.az.xlf

```diff
@@ -14,6 +14,126 @@
             </trans-unit>
+            <trans-unit id="99">
+                <source>This value is not a valid HTML5 color.</source>
+                <target>Bu dəyər doğru bir HTML5 rəngi deyil.</target>
+            </trans-unit>
+            <trans-unit id="100">
+                <source>Please enter a valid birthdate.</source>
+                <target>Zəhmət olmasa doğru bir doğum günü daxil edin.</target>
+            </trans-unit>
+            <trans-unit id="101">
+                <source>The selected choice is invalid.</source>
+                <target>Seçilmiş seçim doğru deyil.</target>
+            </trans-unit>
+            <trans-unit id="102">
+                <source>The collection is invalid.</source>
+                <target>Kolleksiya doğru deyil.</target>
+            </trans-unit>
+            <trans-unit id="103">
+                <source>Please select a valid color.</source>
+                <target>Zəhmət olmasa doğru bir rəng seçin.</target>
+            </trans-unit>
+            <trans-unit id="104">
+                <source>Please select a valid country.</source>
+                <target>Zəhmət olmasa doğru bir ölkə seçin.</target>
+            </trans-unit>
+            <trans-unit id="105">
+                <source>Please select a valid currency.</source>
+                <target>Zəhmət olmasa doğru bir valyuta seçin.</target>
+            </trans-unit>
+            <trans-unit id="106">
+                <source>Please choose a valid date interval.</source>
+                <target>Zəhmət olmasa doğru bir tarix aralığı seçin.</target>
+            </trans-unit>
+            <trans-unit id="107">
+                <source>Please enter a valid date and time.</source>
+                <target>Zəhmət olmasa doğru bir tarix ve saat daxil edin.</target>
+            </trans-unit>
+            <trans-unit id="108">
+                <source>Please enter a valid date.</source>
+                <target>Zəhmət olmasa doğru bir tarix daxil edin.</target>
+            </trans-unit>
+            <trans-unit id="109">
+                <source>Please select a valid file.</source>
+                <target>Zəhmət olmasa doğru bir fayl seçin.</target>
+            </trans-unit>
+            <trans-unit id="110">
+                <source>The hidden field is invalid.</source>
+                <target>Gizli sahə doğru deyil.</target>
+            </trans-unit>
+            <trans-unit id="111">
+                <source>Please enter an integer.</source>
+                <target>Zəhmət olmasa bir tam ədəd daxil edin.</target>
+            </trans-unit>
+            <trans-unit id="112">
+                <source>Please select a valid language.</source>
+                <target>Zəhmət olmasa doğru bir dil seçin.</target>
+            </trans-unit>
+            <trans-unit id="113">
+                <source>Please select a valid locale.</source>
+                <target>Zəhmət olmasa doğru bir dil seçin.</target>
```

Comment:
There is a difference between "language", ie "English", "French" and "Swedish". Compared to a "locale" which is "en", "fr" and "sv". 

Is there a better (maybe more technical) word for "locale"?

Reply:
@Nyholm you are right about this. Changed the word with the more accurate one.

Reply:
Thank you